packages/wrangler/src/type-generation.ts

```diff
@@ -305,8 +305,8 @@ function writeDTSFile({
 
 	let combinedTypeStrings = "";
 	if (formatType === "modules") {
-		combinedTypeStrings += `interface ${envInterface} {\n${envTypeStructure
-			.map((value) => `\t${value}`)
+		combinedTypeStrings += `interface ${envInterface} {${envTypeStructure
+			.map((value, i) => `${i === 0 ? "\n" : ""}\t${value}`)
 			.join("\n")}\n}\n${modulesTypeStructure.join("\n")}`;
```

Comment:
NIT: I think this could be simpler. You are just trying to ensure that every item has a `\n` at the start, no?

```suggestion
			.map((value) => `\n\t${value}`)
			.join("")}\n}\n${modulesTypeStructure.join("\n")}`;
```

Reply:
good point! 😅👍